### PR TITLE
Update hello-hybrid-cert-trust-prereqs.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3713
Added note.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-prereqs.md [hunk 1]
```diff
@@ -112,6 +112,9 @@ Organizations wanting to deploy hybrid certificate trust need their domain joine
  
 Hybrid certificate trust deployments need the device write back feature.  Authentication to the Windows Server 2016 Active Directory Federation Services needs both the user and the computer to authenticate. Typically the users are synchronized, but not devices.  This prevents AD FS from authenticating the computer and results in Windows Hello for Business certificate enrollment failures.  For this reason, Windows Hello for Business deployments need device writeback, which is an Azure Active Directory premium feature.
 
+> [!NOTE]
+> Windows Hello for Business is tied between a user and a device. Both the user and device need to be synchronized between Azure Active Directory and Active Directory, and therefore the device write back is used to update the msDS-KeyCredentialLink on the computer object.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-prereqs.md]:
```suggestion
> Windows Hello for Business is tied between a user and a device. Both the user and device need to be synchronized between Azure Active Directory and Active Directory, and therefore the device writeback is used to update the msDS-KeyCredentialLink on the computer object.
```

**Comment by e0i** [comment]
@dulcemontemayor Would you be able to review and merge this PR based on issue #3713 ? Thank you.

**Comment by e0i** [comment]
@DulceMontemayor Following up for updates.

**Comment by e0i** [comment]
@DulceMontemayor

Following up for updates.

cc @dansimp

**Comment by e0i** [comment]
@DulceMontemayor

Following up for updates.

cc @Dansimp

**Pull request merged**